chevah/compat/nt_filesystem.py

```diff
@@ -262,6 +262,8 @@ def getSegmentsFromRealPath(self, path):
         if path is None or path == u'':
             return segments
 
+        path = text_type(path)
```

Comment:
This ensures the resulting segments are Unicode and not bytes, using `six`.

Reply:
I think that the current code is ok.